rust/src/offer.rs

```diff
@@ -23,3 +25,31 @@ pub async fn get_offer() -> Result<Offer> {
         .await
         .map_err(|e| anyhow!("Failed to fetch offer {e:?}"))
 }
+
+#[derive(Debug, Deserialize, PartialEq, Eq)]
+#[serde(rename_all = "camelCase")]
+pub struct FeeRecommendation {
+    pub fastest_fee: u32,
+    pub half_hour_fee: u32,
+    pub hour_fee: u32,
+    pub minimum_fee: u32,
+}
+
+pub async fn get_fee_recommendation(network: Network) -> Result<FeeRecommendation> {
```

Comment:
🛠️ We have the possibility to use Electrum for this! I added this before:

```
let estimated_fee_rate = self
            .lightning
            .wallet
            .estimate_fee(ConfirmationTarget::Normal)
            .map_err(|_| anyhow!("Failed to estimate fee"))?;
```

I think a dependency on mempool is not needed here, and just makes everything even more brittle! we cannot easily know if we hit quotas with their API.

Reply:
Done!